includes/admin/load.php

```diff
@@ -393,3 +399,39 @@ static function ( $name ) {
 		);
 	}
 }
+
+/**
+ * Callback function that print plugin progress indicator script.
+ *
+ * @since n.e.x.t
+ */
+function perflab_print_plugin_progress_indicator_script() {
+	$js_function = <<<JS
+		function addPluginProgressIndicator( message ) {
+			document.addEventListener( 'DOMContentLoaded', function () {
+				document.addEventListener( 'click', function ( event ) {
+					if (
+						event.target.classList.contains(
+							'perflab-install-active-plugin'
+						)
+					) {
+						const target = event.target;
+						target.classList.add( 'updating-message' );
+						target.textContent = message;
+
+						wp.a11y.speak(message);
+					}
+				} );
+			} );
+		}
+JS;
+
+	wp_print_inline_script_tag(
+		sprintf(
+			'( %s )( %s );',
+			$js_function,
+			wp_json_encode( __( 'Activating…', 'default' ) )
```

Comment:
I just realized that this is `'Activating…'` when the core string appears to rather be `'Activating...'`:

https://github.com/WordPress/wordpress-develop/blob/f9e5c6fb22a331746a1821e393c9ea6a6e6d70a0/src/js/_enqueues/wp/updates.js#L1090

Does this need to be amended?